content/en/real_user_monitoring/session_replay/browser/_index.md

```diff
@@ -28,11 +28,11 @@ Session Replay expands your user experience monitoring by allowing you to captur
 
 The RUM Browser SDK is [open source][1] and leverages the open source [rrweb][2] project.
 
-## Session Replay recorder
+## How the Session Replay recorder works
 
 The Session Replay recorder is part of the RUM Browser SDK. The recorder takes a snapshot of the browser's DOM and CSS by tailing and recording events happening on a web page (such as DOM modification, mouse move, clicks, and input events) along with these events' timestamps.
 
-Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view. Session Replay follows the same 30 day retention policy as normal RUM sessions.
+Datadog then rebuilds the web page and re-applies the recorded events at the appropriate time in the replay view.
```

Comment:
@rtrieu I changed some stuff here, but for some reason GitHub wouldn't let me do a suggested changed so I went ahead and committed directly. Feel free to revert if you disagree :)